src/dpsvalidationservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/dpsvalidationservice/dfcsm/ValidationController.java

```diff
@@ -85,14 +85,16 @@ public String getValidOpenDFCMCase(@ApiParam(value = "driversLicense", required
      * @return a String with media type application/xml
      */
     @ExceptionHandler(MissingServletRequestParameterException.class)
-    public ResponseEntity<String> handleMissingParams(MissingServletRequestParameterException ex) {
+    public ResponseEntity<GetValidOpenDFCMCase> handleMissingParams(MissingServletRequestParameterException ex) {
+
         String paramName = ex.getParameterName();
-        logger.fatal("Exception in  : ValidationController " + ex.getMessage());
+        logger.error("Exception in  : ValidationController " + ex.getMessage());
```

Comment:
@shaunmillargov do we need to print a stack trace here? I remember in previous commits there were logs with an exception message as well as a stack trace printed. I might be behind on this.

Reply:
Yes, I would recommend adding the stack trace too. Order 1.) log the error, 2.) print a stack trace. This will catch errors in the situation when logging has not been set up properly. It's the way we always did it for the AG.

Reply:
Good catch, updated